tools/src/main/scala/org/apache/spark/sql/rapids/tool/profiling/ApplicationInfo.scala

```diff
@@ -579,49 +579,75 @@ class ApplicationInfo(
     if (allDataFrames.contains(s"blockManagersDF_$index") &&
         allDataFrames.contains(s"resourceProfilesDF_$index")) {
 
-      s"""select $index as appIndex, e.executorID, e.totalCores,
-         |b.maxMem, b.maxOnHeapMem,b.maxOffHeapMem,
-         |r.exec_cpu, r.exec_mem, r.exec_gpu, r.exec_offheap, r.task_cpu, r.task_gpu
+      s"""select $index as appIndex,
+         |first(r.id) as resourceProfileId,
+         |count(e.executorID) as numExecutors,
```

Comment:
this isn't unique per resource profile id, which we want

Reply:
Good catch. I was not testing this on event logs with mutliple resource ids. Updated the query which may not be efficient but works as intended. My SQL skills isn't that good. Please take a look at this.